spec/index.bs

```diff
@@ -1350,7 +1350,7 @@ For more information on issuing CORS requests for image and video elements, cons
 WebGPU does not provide color management. All values within WebGPU (such as texture elements)
 are raw numeric values, not color-managed color values.
 
-WebGPU *does* interface with color-managed outputs (via {{GPUPresentationConfiguration}}) and inputs
+WebGPU *does* interface with color-managed outputs (via {{GPUContextConfiguration}}) and inputs
```

Comment:
`GPUContextConfiguration` is a red flag. It sounds like something that would configure GPU in general.